experiments/tf_trainer/tf_word_label_embedding/run.py

```diff
@@ -21,28 +21,17 @@
 tf.app.flags.DEFINE_string('embeddings_path',
                            'local_data/glove.6B/glove.6B.100d.txt',
                            'Path to the embeddings file.')
-tf.app.flags.DEFINE_integer('batch_size', 64,
-                            'The batch size to use during training.')
-tf.app.flags.DEFINE_integer('train_steps', 30000,
-                            'The number of steps to train for.')
-tf.app.flags.DEFINE_integer('eval_period', 500,
-                            'The number of steps per eval period.')
-tf.app.flags.DEFINE_integer('eval_steps', 50,
-                            'The number of steps to eval for.')
 
 
 def main(argv):
   del argv  # unused
 
-  embeddings_path = FLAGS.embeddings_path
-
-  preprocessor = text_preprocessor.TextPreprocessor(embeddings_path)
+  preprocessor = text_preprocessor.TextPreprocessor(FLAGS.embeddings_path)
 
   nltk.download('punkt')
   train_preprocess_fn = preprocessor.train_preprocess_fn(nltk.word_tokenize)
   dataset = tfrecord_input.TFRecordInputWithTokenizer(
       train_preprocess_fn=train_preprocess_fn,
-      batch_size=FLAGS.batch_size,
       max_seq_len=5000)
```

Comment:
Should be set as a flag. Or at least should not be overwriting a default value from https://github.com/conversationai/conversationai-models/blob/cb769b8a786b248cde6ba3f7e309a7a14dc05a47/experiments/tf_trainer/common/tfrecord_input.py#L146

Reply:
Will address max_seq_len in a future change.